helpers/validator/validator.js

```diff
@@ -161,25 +174,29 @@ Validator.prototype.validate = function(value, rules, callback) {
 };
 
 /**
- * Validator field constructor
+ * Validator field constructor.
+ *
  * @type {Field}
  */
 Validator.prototype.Field = Field;
 
 /**
- * Create field instance
- * @param {string|string[]} path Field path
- * @param {*} value Validated value
- * @param {Object} rules Rule set
- * @param {*=} thisArg This reference for Validation methods. Optional
+ * Create field instance.
+ *
+ * @param {string|string[]} path - Field path
+ * @param {*} value - Validated value
+ * @param {Object} rules - Rule set
+ * @param {*=} thisArg - This reference for Validation methods. Optional
```

Comment:
What's this type?

Reply:
From JSDoc: `Indicates that the parameter is optional`.

This is similar to `{?*}`, but only used for optional params in JSDoc.